packages/@ember/-internals/meta/lib/meta.ts

```diff
@@ -480,16 +494,15 @@ export class Meta {
     while (pointer !== null) {
       let map = pointer._descriptors;
       if (map !== undefined) {
-        for (let key in map) {
+        map.forEach((value, key) => {
```

Comment:
Note: I use `Map.forEach()` for the iteration, since it is the only method that works with IE11.